geometry_msgs/Point Position 
geometry_msgs/Point Velocity 
bool xEnabled
bool yEnabled 
bool thetaEnabled 
